### PR TITLE
Tutorial example for classification

### LINKED ISSUE #62: Creating a tutorial example for classification
Hi, similar to my other issue #61 , I wanted to propose adding a GP classification example. It can fit into Non-Gaussian Likelihoods or a different page.

I'd written a tutorial for Pyro: https://github.com/prog-ml/prog-ml.github.io/blob/main/notebooks/gaussian_processes/pyro-binary-classification.ipynb

that can be easily adapted for tinyGP. 

Please let me know if this is of interest. My students or I could create a PR for this.

### PR DESCRIPTION
**Pull request by yadav-sachin**
closes #62

Adding an example for classification task using GP.
Here I have used a simple binary classification task on XOR Dataset for demonstration.

This tutorial is adapted from: https://github.com/prog-ml/prog-ml.github.io/blob/main/notebooks/gaussian_processes/pyro-binary-classification.ipynb

### CODE REVIEW

File: docs/tutorials/gp-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
```
Comment (dfm) [inline docs/tutorials/gp-classification.ipynb:6]: This should include a `hide-cell` tag. Take a look at some of the other tutorials.

File: docs/tutorials/gp-classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+   "source": [
+    "(gp-classification)="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## GP for Classification"
```
Comment (dfm) [inline docs/tutorials/gp-classification.ipynb:37]: This should be `# GP for Classification` to get the build to work.

Comment (dfm) [reply docs/tutorials/gp-classification.ipynb:37]: And maybe we want to think about this title a bit... maybe just `# Classification` would be better.

**Review by dfm — CHANGES_REQUESTED** [review]

File: docs/tutorials/gp-classification.ipynb (hunk 2 above)
Comment (yadav-sachin) [reply docs/tutorials/gp-classification.ipynb]: What is causing the docs build to fail now? 🤔 
https://readthedocs.org/projects/tinygp/builds/17030342/

Comment (dfm) [reply docs/tutorials/gp-classification.ipynb]: Your branch is quite out of date. You should merge the current main branch into yours to fix the build, I think.

File: docs/tutorials/classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+    "    if y is not None:\n",
+    "        # Posterior Inference on true_X input values\n",
+    "        numpyro.deterministic(\"pred\", gp.condition(gp_out, true_X).gp.loc)\n",
+    "\n",
+    "\n",
+    "nuts_kernel = numpyro.infer.NUTS(model, target_accept_prob=0.8)\n",
+    "mcmc = numpyro.infer.MCMC(\n",
+    "    nuts_kernel,\n",
+    "    num_warmup=1000,\n",
+    "    num_samples=1000,\n",
```
Comment (yadav-sachin) [inline docs/tutorials/classification.ipynb:214]: It's currently taking too long to build.
Should I reduce the num_samples, and num_warmup (to 500)?

**Comment by yadav-sachin** [comment]
Waiting for initial review @dfm

**Comment by yadav-sachin** [comment]
Closing because of inactivity

**Pull request closed without merging**